src/DeepLinkResources/Icon.php

```diff
@@ -18,6 +20,15 @@ public static function new(string $url, int $width, int $height): self
         return new Icon($url, $width, $height);
     }
 
+    public function getArray(): array
```

Comment:
I'm guessing the `getArray()` is just how the model is returned? I was wondering why this was named so generically.

Reply:
Correct. I need something that builds the potential array of properties before sending in through the various concerns for processing. I settled on `getArray()` but am open to suggestions for a better way to name this.

Reply:
I think this is fine, I could understand what it was doing by just looking at the function.